app/views/patients/_fulfillment.html.erb

```diff
@@ -29,10 +29,11 @@
                           options_for_select(weeks_options, patient.fulfillment.gestation_at_procedure),
                           label: 'Weeks along at procedure',
                           autocomplete: 'off' %>
-            <%= pt.text_field :procedure_date,
+            <%= pt.date_field :procedure_date,
+                                  value: patient.procedure_date_display,
                                   label: 'Procedure date',
-                                  autocomplete: 'off', format: '%Y-%m-%d',
-                                  placeholder: 'YYYY-MM-DD' %>
+                                  autocomplete: 'off'
+                                   %>
```

Comment:
I usually stay away from style but it would be A+ if you could put this closing tag back on the same line as `autocomplete`. Keeps our views a little cleaner that way, and they're one of the toughest parts of this app to read as engineers.